backends/blocked/ceed-blocked.c

```diff
@@ -27,9 +26,6 @@ static int CeedInit_Blocked(const char *resource, Ceed ceed) {
   CeedCallBackend(CeedInit("/cpu/self/ref/serial", &ceed_ref));
   CeedCallBackend(CeedSetDelegate(ceed, ceed_ref));
 
-  // Set fallback Ceed resource for advanced operator functionality
-  CeedCallBackend(CeedSetOperatorFallbackResource(ceed, fallback_resource));
```

Comment:
Note - in none of these 3 backends does the `/cpu/self/ref/serial` backend impliment any preconditioning related functionality that these backends don't implement themselves, so falling back and creating the cloned operator is just a waste of memory.

Reply:
Isn't this needed for `CeedOperatorLinearAssembleQFunction_Blocked` and `CeedOperatorLinearAssembleQFunction_Opt`?